### PR TITLE
combined webpack configs and added prod scripts; js includes in index…

### LINKED ISSUE #66: Combine dev + prod webpack config files

### PR DESCRIPTION
**Pull request by skomputer**
….html depend on environment

Closes #66.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,9 +14,13 @@
     "dev": "npm run dev-serve | npm run dev-build",
     "dev-serve": "NODE_ENV=development node server/index.js",
     "dev-build": "webpack-dev-server --progress --history-api-fallback --hot --content-base build --port 8090 --config webpack.dev.config",
+    "prod": "npm run prod-serve | npm run prod-build",
```
Comment (aslagle) [inline package.json]: should prod-build come first? it should exit when it's finished, unlike dev-build.

File: webpack.prod.config.js [hunk 2]
```diff
@@ -1,37 +1,17 @@
...
-  },
-  resolve: {
-    extensions: ["", ".webpack.js", ".web.js", ".js", ".ts", ".tsx"]
-  }
+config.entry.app = [
+  "./src/index.ts"
+];
+
+config.output = {
+  path: "./dist",
```
Comment (aslagle) [inline webpack.prod.config.js]: why not output into the server's public if that's where we're using it from?

**Comment by aslagle** [comment]
you didn't end up needing different loaders for dev and prod?

File: package.json (hunk 1 above)
Comment (skomputer) [inline package.json]: i'm not seeing any difference functionally, but running prod-build first ends up hiding a bunch of output from the build...

Comment (aslagle) [inline package.json]: yeah, and running prod-serve first hides output from the server. could you do npm run prod-build && npm run prod-serve?

**Comment by skomputer** [comment]
like which loaders?

File: package.json (hunk 1 above)
Comment (skomputer) [inline package.json]: :+1:

File: webpack.prod.config.js [hunk 3]
```diff
@@ -13,20 +12,22 @@ var config = {
...
     loaders: [
-      { 
+      {
         test: /\.tsx?$/,
         exclude: [/node_modules/],
         loaders: [
-          'react-hot', 
-          'ts-loader' 
-        ] 
+          'ts-loader'
```
Comment (aslagle) [inline webpack.prod.config.js]: you still need this?

**Comment by aslagle** [comment]
:+1:

**Pull request merged**